src/plugins/crypto/testmod_crypto.c

```diff
@@ -213,7 +213,7 @@ static void test_enc_and_dec_with_null()
 {
 	elektraCryptoHandle *handle;
 	KeySet *config;
-	Key *errorKey = keyNew(KEY_END);
+	Key *errorKey = keyNew ("", KEY_END);
```

Comment:
speaking of code formatting. do we have a decision for blanks and brackets, etc.?